demisto_sdk/commands/lint/linter.py

```diff
@@ -286,9 +298,24 @@ def _gather_facts(self, modules: dict) -> bool:
         # Docker images
         if self._facts["docker_engine"]:
             logger.info(f"{log_prompt} - Collecting all docker images to pull")
-            self._facts["images"] = [
-                [image, -1] for image in get_all_docker_images(script_obj=script_obj)
-            ]
+            yml_obj_id = (
+                yml_obj.get("commonfields", {}).get("id", "")
+                if isinstance(yml_obj, dict)
+                else ""
+            )
+            images = self._get_docker_images_for_lint(
+                script_obj=script_obj,
+                script_id=yml_obj_id,
+                docker_image_flag=self.docker_image,
+            )
+            if not images:
+                # If no docker images to run on - skip checks in both docker and host
+                logger.info(
+                    f"{log_prompt} - No docker images to run on - Skipping run lint in host as well."
+                )
+                return True
+            self._facts["images"] = [[image, -1] for image in images]
```

Comment:
what's this `-1` meaning?

Reply:
that was there from before. Also not sure what it does though

Reply:
As Judah said, it was there before... the -1 will be replaced with the python version (that is extracted from the docker image) - you can see it in the loop that starts in line: 338.